labs/firepower-restapi-104/2.md

```diff
@@ -1,62 +1,35 @@
 # Response structure
 The response structure can be:
 
--   `LOCATION` – Newly created resource ID; for POST only—holds the new resource
-    ID (as a URI representation).
+-   `LOCATION` – newly created resource ID; for POST only—holds the new resource ID (as a URI representation).
 
--   `CONTENT-TYPE` – Media type describing the response message body; describes
-    the representation and syntax of the response message body.
+-   `CONTENT-TYPE` – media type describing the response message body. It describes the representation and syntax of the response message body.
 
--   Each response includes an HTTP status or error code. Available codes fall
-    into these categories:
+-   Each response includes an HTTP status or error code. Available codes fall into these categories:
 
--   20x - A two-hundred series code indicates successful operation, including:
+  -   20x - A two-hundred series code indicates successful operation, including:
 
-    -   **200 OK**
+      -   **200 OK** – standard response for successful requests.
```

Comment:
This capitalization (Standard response...) is correct based on Cisco Style: https://www-author3.cisco.com/c/en/us/td/docs/general/style/guide/Latest/stylegd/elements.html#28646 We do not use em dashes, however. So this list can remain as-is.

Reply:
Oh, I thought you said we do use em dashes. I changed tons of dashes to em dashes in lots of docs. Do you mean just in a bullet list like this one we use a regular dash? I will put this list back to the original style (which is what I think you mean). I'll have lots of other docs to do this in.